django/core/management/base.py

```diff
@@ -303,7 +303,11 @@ def run_from_argv(self, argv):
                 self.stderr.write('%s: %s' % (e.__class__.__name__, e))
             sys.exit(1)
         finally:
-            connections.close_all()
+            try:
+                connections.close_all()
+            except ImproperlyConfigured:
+                # Don't moan if connections aren't setup at this point
```

Comment:
Don't moan -> Ignore
Also, maybe explain under what circumstances that happens?